docs/benchmark.md

```diff
@@ -0,0 +1,98 @@
+# Benchmarks
+
+## Running the benchmark
+
+You can run the benchmark by executing `./gradlew :android-benchmark:benchmark:connectedCheck`. (Note that the benchmark should be run on an actual device)
+You can then take the output from the benchmark and run it through `DataParser.kt` to get a structured breakdown as seen below.
```

Comment:
Let's add a note here about paying attention to units more than percentages

Reply:
bf05483